src/Trixi.jl

```diff
@@ -253,6 +253,8 @@ function __init__()
 
   init_p4est()
 
+  register_error_hints()
```

Comment:
Did you check the impact on `using` time?

Reply:
I ran `@time using Trixi` with this PR as-is and with this PR but with the call to `register_error_hints()` commented out. Over ten trials, the minimum time for `using Trixi` increased on my M1 Macbook by 0.63% and decreased on Roci by 0.01%. I'd say both numbers are well within error margins.

Reply:
Thanks!